packages/gamut-labs/src/Avatar/index.tsx

```diff
@@ -99,10 +94,6 @@ export const Avatar: React.FC<AvatarProps> = ({
     disableDropshadow={disableDropshadow}
     data-testid="avatar-container"
   >
-    <Image
-      width={avatarSizes[size]}
-      height={avatarSizes[size]}
-      {...avatarImageProps}
-    />
+    <Image width={`${size}px`} height={`${size}px`} {...avatarImageProps} />
```

Comment:
The shorthand for both `width` and `height` together is annoyingly `dimensions` because of other prop conflicts for buttons etc but it also will do all of the transformations for you so we can just pass numbers here.

```suggestion
    <Image dimensions={size} {...avatarImageProps} />
```